### ISSUE TITLE
Create an easy-to-use api for DIM-related actions

### ISSUE THREAD

**Issue by awesomeness01** [DestinyItemManager/DIM#5382]
**Proposed change**
I would like to see an API added to DIM that would allow people to access DIM from their own apps, for example switching loadouts to a saved one in DIM. This would allow people to write apps that interface with other technologies, like voice control software (e.g. Google Assistant, Alexa)

**How does this fit into your workflow?**
Being on console makes it annoying to have to open my inventory every time and wait for it to load, so I would love to just be able to say something like "OK Google, tell Destiny to switch to my PVP loadout". Of course it would be up to other developers to write a skill for Google Assistant or Alexa, unless you develop that too.

**Comment by bhollis** [comment]
Good idea! We in fact already have this planned, but have some other features ahead of it in priority.
